docs/telemetry.rst

```diff
@@ -67,6 +67,15 @@ gc
 
 The ``gc`` telemetry device enables GC logs for the benchmark candidate. You can use tools like `GCViewer <https://github.com/chewiebug/GCViewer>`_ to analyze the GC logs.
 
+If the runtime JDK is Java 9 or better, the following telemetry parameters can be specified:
```

Comment:
I think `or higher` or `or greater` makes more sense than `or better` here.

Reply:
Thanks! I'll change this before I merge.

Reply:
Addressed in https://github.com/elastic/rally/pull/986/commits/e4deee50ecb87bf502c6716596e5a3471a8dc2d8.